neonvm/controllers/alerttracker/tracker.go

```diff
@@ -0,0 +1,87 @@
+package alerttracker
```

Comment:
not sure how I feel about this package name. Strictly speaking, we're not tracking _alerts_... I can't immediately think of a better name though...

Reply:
Ah well. Can bikeshed the name at a later date haha :)